posts/2024-10-08-24.0.0.10.adoc

```diff
@@ -0,0 +1,172 @@
+
+Here are the link:https://openjdk.org/projects/jdk/23/[JEP changes in Java 23]:
+
+* 455: link:https://openjdk.org/jeps/455[Primitive Types in Patterns, instanceof, and switch (Preview)]
+* 466: link:https://openjdk.org/jeps/466[Class-File API (Second Preview)]
+* 467: link:https://openjdk.org/jeps/467[Markdown Documentation Comments]
+* 469: link:https://openjdk.org/jeps/469[Vector API (Eighth Incubator)]
+* 471: link:https://openjdk.org/jeps/471[Deprecate the Memory-Access Methods in sun.misc.Unsafe for Removal]
+* 473: link:https://openjdk.org/jeps/473[Stream Gatherers (Second Preview)]
+* 474: link:https://openjdk.org/jeps/474[ZGC: Generational Mode by Default]
+* 476: link:https://openjdk.org/jeps/476[Module Import Declarations (Preview)]
+* 477: link:https://openjdk.org/jeps/477[Implicitly Declared Classes and Instance Main Methods (Third Preview)]
+* 480: link:https://openjdk.org/jeps/480[Structured Concurrency (Third Preview)]
+* 481: link:https://openjdk.org/jeps/481[Scoped Values (Third Preview)]
+* 482: link:https://openjdk.org/jeps/482[Flexible Constructor Bodies (Second Preview)]
+
+Take advantage of the new changes in Java 23 in Open Liberty now and get more time to review your applications, microservices and runtime environments on your favorite server runtime!
+
+To start using Java 23 with Open Liberty:
+
+. Download the link:https://developer.ibm.com/languages/java/semeru-runtimes/downloads/?version=23[latest release of Java 23]
+. Download and install the link:{url-prefix}/downloads/#runtime_releases[24.0.0.10] version of Open Liberty
+. Edit your Liberty link:{url-prefix}/docs/latest/reference/config/server-configuration-overview.html#server-env[server.env file] to point `JAVA_HOME` to your Java 23 installation
+. Start testing today!
+
+
+For more information on Open Liberty, please visit our link:{url-prefix}/docs[documentation page].
+
+For more information about Java 23, see the following resources:
+
+- link:https://jdk.java.net/23/release-notes[Release notes]
+- link:https://docs.oracle.com/en/java/javase/23/docs/api/index.html[Javadocs]
+- link:https://docs.oracle.com/en/java/javase/23/migrate/index.html[Migration guide]
+- link:https://developer.ibm.com/languages/java/semeru-runtimes/downloads/?version=23[Download page]
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/29749
+// Contact/Reviewer: jhanders34
+// // // // // // // //
+[#mp]
+== Versionless features for standalone MicroProfile Liberty features
```

Comment:
```suggestion
== Versionless stand-alone MicroProfile Liberty features
```
Update to match the bulleted list above?  If you keep the current title `stand-alone` needs to be updated.